gl_webgl.go

```diff
@@ -352,19 +374,29 @@ func GetString(pname Enum) string {
 }
 
 func GetTexParameterfv(dst []float32, target, pname Enum) {
-	panic("GetTexParameterfv: not yet implemented")
+	dst[0] = float32(c.Call("getTexParameter", pname).Float())
```

Comment:
I'm not sure how to correctly handle situations when `len(dst) > 1`, but I think in practice that's never going to be the case at this time.

It's just an OpenGL API quirk that's more apparent with Go's type system.